tests/test_library_integration.py

```
@@ -0,0 +1,111 @@
+# Mark all tests in this module as requiring network access
+pytestmark = pytest.mark.network
+
+
+class TestRealLibraryFetch:
+    """Integration tests that fetch from the real library."""
+
+    def test_fetch_real_index(self):
+        """Test fetching the real index.json from GitHub."""
+        with tempfile.TemporaryDirectory() as tmpdir:
+            client = LibraryClient(cache_dir=Path(tmpdir))
+            index, from_cache = client.fetch_index(no_cache=True)
```

Ruff (0.14.14)

[warning] Unpacked variable `from_cache` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)